app/javascript/src/components/Dashboard/Notes/Note.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+
+import { Clock, MenuVertical } from "@bigbinary/neeto-icons";
+import { Typography, Tag, Avatar, Tooltip, Dropdown } from "neetoui";
```

Comment:
Why have you followed two different module reference patterns? Like for neeti-icons you have prefix with `@bigbinary`, but for `neetoui` you haven't.

Ideally you don't need to prefix `@bigbinary`. Add necessary aliases.